src/window/build_menu.c

```diff
@@ -202,38 +220,42 @@ static void button_menu_index(int param1, int param2)
     button_menu_item(button_index_to_submenu_item(param1 - 1));
 }
 
+static int set_submenu_for_type(building_type type)
+{
+    building_menu_group current_menu = data.selected_submenu;
+    switch (type) {
+    case BUILDING_MENU_FARMS:
```

Comment:
Indent everything within the switch statement please.

Reply:
Done already while you were reviewing, sorry.